examples/analog_out/cont_gen_voltage_wfm_int_clk.py

```diff
@@ -0,0 +1,32 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a continuous periodic
+waveform using an internal sample clock.
+"""
+
+import numpy as np
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    frequency = 10
+    amplitude = 1
+    sampling_rate = 100
+    duration = 1
+
+    # generate the time array
+    t = np.arange(0, duration, 1 / sampling_rate)
+    # Generate the sine wave
+    data = amplitude * np.sin(2 * np.pi * frequency * t)
```

Comment:
[style] Instead of declaring a ton of global variables, how about writing a function that takes those variables as parameters?

for example,
```
def create_sine_wave(frequency: float, amplitude: float, sampling_rate: float, duration: float) -> numpy.typing.NDArray[numpy.double]:
   ...
```

Reply:
Created helper to for the function.